src/components/ProductCard/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Link from 'next/link'
+
+const ProductCard = () => {
+  return (
+    <div className="product-card">
+      <Link href="/product-view">
+        <a className="product-card__link">
+          <div className="product-card__content">
+            <span>
+              Click Me!
+            </span>
```

Comment:
How about one line instead?
```suggestion
            <span>Click Me!</span>
```